### PR TITLE
Link to operator as the default k8s tutorial

### PR DESCRIPTION
**Pull request by sougou**
Signed-off-by: Sugu Sougoumarane <ssougou@gmail.com>

### CODE REVIEW

**Comment by sougou** [comment]
Ref https://github.com/vitessio/vitess/issues/6439

File: content/en/docs/get-started/helm.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 ---
-title: Kubernetes Helm Chart
-weight: 3
+title: Helm Chart (deprecated)
+weight: 4
```
Comment (deepthi) [inline content/en/docs/get-started/helm.md]:
```suggestion
weight: 5
```

File: content/en/docs/get-started/operator.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
 ---
 title: Vitess Operator for Kubernetes
-weight: 3
+weight: 1
```
Comment (deepthi) [inline content/en/docs/get-started/operator.md]:
```suggestion
weight: 3
```

**Review by deepthi — COMMENTED** [review]
This moved Operator above "What is Vitess" and "Architecture". I believe those should still be the top items (and possibly also "Supported Databases" before the tutorials).

**Review by deepthi — APPROVED** [review]
:+1:

**Pull request merged**